doc/orchestrator.rst

```diff
@@ -71,100 +335,217 @@ This method is deemed ``locality based inference`` since data is local to each
+.. note::
+  Change the `exe_args` argument to the path of the application script
+  on your file system to run the example.
 
-A full example of configuring KeyDB to run in SmartSim is shown below
+Use the ``RunSettings`` helper functions to
+configure the the distribution of computational tasks (``RunSettings.set_nodes()``). In this
+example, we specify to SmartSim that we intend the Model to run on a single compute node.
 
-.. code-block:: bash
+.. code-block:: python
 
-  # build KeyDB
-  # see https://github.com/EQ-Alpha/KeyDB
+    # Initialize a RunSettings object
+    model_settings = exp.create_run_settings(exe=exe_ex, exe_args="/lus/scratch/richaama/clustered_model.py")
+    # Configure RunSettings object
+    model_settings.set_nodes(1)
 
-  # get KeyDB configuration file
-  wget https://github.com/CrayLabs/SmartSim/blob/d3d252b611c9ce9d9429ba6eeb71c15471a78f08/smartsim/_core/config/keydb.conf
+Step 2: Initialize
+""""""""""""""""""
+Next, create a ``Model`` instance using the ``Experiment.create_model()``.
+Pass the ``model_settings`` object as an argument
+to the ``create_model()`` function and assign to the variable ``model``.
+.. code-block:: python
+
+    # Initialize a SmartSim Model
+    model = exp.create_model("colo_model", model_settings)
 
-  export REDIS_PATH=/path/to/keydb-server
-  export REDIS_CONF=/path/to/keydb.conf
+Step 2: Colocate
+""""""""""""""""
+To colocate the model, use the ``Model.colocate_db_uds()`` function.
+This function will colocate an Orchestrator instance with this Model over
+a Unix domain socket connection.
+.. code-block:: python
+
+    # Colocate the Model
+    model.colocate_db_tcp()
+
+Step 3: Start
+"""""""""""""
+Next, launch the colocated model instance using the ``Experiment.start()`` function.
+.. code-block:: python
+
+    # Launch the colocated Model
+    exp.start(model, block=True, summary=True)
+
+test
+
+Cleanup Experiment
+------------------
+
+.. code-block:: python
+
+    logger.info(exp.summary())
+
+When you run the experiment, the following output will appear::
 
-  # run smartsim workload
+  |    | Name   | Entity-Type   | JobID     | RunID   | Time    | Status    | Returncode   |
+  |----|--------|---------------|-----------|---------|---------|-----------|--------------|
+  | 0  | model  | Model         | 1592652.0 | 0       | 10.1039 | Completed | 0            |
+
+======================
+Multiple Orchestrators
+======================
+SmartSim supports automating the deployment of multiple Orchestrators
+from within an Experiment. Data communication for all
+Communication with the database via a SmartRedis Client is possible from the
+`db_identifier` argument that required when initializing an Orchestrator or
+colocated Model during a multi database Experiment. When initializing a SmartRedis
+client during the Experiment, first create a ``ConfigOptions`` object
+with the `db_identifier` argument created during before passing object to the Client()
```

Comment:
wonky: "argument created during before passing object to the..."

Consider editing this whole paragraph again. it seems like it got lost at the end of the file!